RoaringBitmap/src/test/java/org/roaringbitmap/buffer/ImmutableRoaringBitmapBatchIteratorTest.java

```diff
@@ -192,4 +194,19 @@ public void testBatchIteratorWithAdvanceIfNeeded() {
         assertEquals(batch[1], (3 << 16) + 5);
         assertEquals(batch[2], (3 << 16) + 10);
     }
+
+    @ParameterizedTest
+    @ValueSource(ints = {10, 11, 12, 13, 14, 15, 18, 20, 21, 23, 24})
+    public void testBatchIteratorWithAdvancedIfNeededWithZeroLengthRun(int number) {
+       MutableRoaringBitmap bitmap = MutableRoaringBitmap.bitmapOf(10, 11, 12, 13, 14, 15, 18, 20, 21, 22, 23, 24);
+       bitmap.runOptimize();
+        BatchIterator it = bitmap.getBatchIterator();
+        it.advanceIfNeeded(number);
+        assertTrue(it.hasNext());
+        int[] batch = new int[10];
+        int n = it.nextBatch(batch);
+        int i = Arrays.binarySearch(batch, 0, n, number);
+        assertTrue(i >= 0, "key " + number + " not found");
+        assertEquals(batch[i], number);
```

Comment:
I don't like making comments like this because it should be automated and I'm not sure why the format check passed, but could you align the indentation with line 202 here please?

Reply:
> I don't like making comments like this because it should be automated and I'm not sure why the format check passed, but could you align the indentation with line 202 here please?

:white_check_mark: